tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -427,27 +427,42 @@ def augmented_dickey_fuller(x, param):
 
     :param x: the time series to calculate the feature of
     :type x: numpy.ndarray
-    :param param: contains dictionaries {"attr": x} with x str, either "teststat", "pvalue" or "usedlag"
+    :param param: contains dictionaries {"attr": x, "autolag": y} with x str, either "teststat", "pvalue" or "usedlag"
+                  and with y str, either of "AIC", "BIC", "t-stats" or None.
```

Comment:
Why can it be "none"? Why would I want to have this?
If this is really a usual case, you might want to document it here :-)

Reply:
I added a reference to StatsModels' documentation there in e77b909.

> Why can it be "none"? Why would I want to have this?

This is corresponding to StatsModels' `adfuller()` function. 

https://www.statsmodels.org/stable/generated/statsmodels.tsa.stattools.adfuller.html

This function accepts `None` as the `autolag` parameter, which means we use the
constant `12*(nobs/100)^{1/4}` as the maximum lag length. This is a sensible choice
of the parameter and especially useful when we want to avoid CPU/memory
intensive fitting process.